src/components/swap/GasBreakdownTooltip.tsx

```diff
@@ -83,7 +91,7 @@ export function GasBreakdownTooltip({
           <Trans>
             <InlineUniswapXGradient>UniswapX</InlineUniswapXGradient> aggregates liquidity sources for better prices and
             gas free swaps.
-          </Trans>{' '}
+          </Trans>
```

Comment:
Why did you remove the space here?

Reply:
I don't use these kind of spaces in my codebases and kinda did it by inertia. Checked and you are right, it served a purpose and looks better with it, so I reverted it.